cameras/paths.csv

```diff
@@ -13,10 +13,14 @@ Axis,camera,P1224,(00:40:8C),rtsp://{{username}}:{{password}}@{{ip_address}}/axi
+Hikvision,nvr,DS-7600 series,(8C:E7:48),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/Streaming/Channels/{{channel}}{{stream}},H.264,554,01;02,1;2;3;4;5;6;7;8,hi-res;lo-res,Channel {{channel}},02,01,admin,12345,,http://www.hikvision.com//uploadfile/image/20150831025213965.PDF
 LTS,camera,CMIP8242WB,(C4:2F:90|C0:56:E3|BC:AD:28|44:19:B6|28:57:BE),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/,H.264,554,,,,,,,admin,12345,,http://www.ltsecurityinc.com/amfilerating/file/download/file_id/1008/
 Digital Watchdog,nvr,VMAX A1,(00:1C:84|3C:04:BF),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/rtsp/unicast/live/profile-{{channel}},H.264,554,,1;2;3;4,,,,,admin,,,http://digital-watchdog.com/forcedown?file_path=_gendownloads/fe8bcd0f-58df-4414-ae3c-bc76fda207c1/&file_name=DW_MAN_VMAX%20A1_Rev0416.pdf
 Digital Watchdog,nvr,VMAX 960H Core,(00:1C:84|3C:04:BF),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/rtsp/unicast/live/profile-{{channel}},H.264,554,,1;2;3;4,,,,,admin,,,http://www.advancealarms.com/user-manuals/digital-watchdog-vmax-960h-core.pdf
 Digital Watchdog,nvr,VMAX IP Plus,(00:1C:84|3C:04:BF),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/rtsp/unicast/live/profile-{{channel}},H.264,554,,1;2;3;4;5;6;7;8;9;10;11;12;13;14;15;16,,,,,admin,,,http://digital-watchdog.com/forcedown?file_path=_gendownloads/1a4b1b60-fee9-4212-9229-5c3e27685431/&file_name=DW_MAN_VMAXIPPlus_Rev0416.pdf
 Amcrest,nvr,NV1104,(90:02:A9),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/cam/realmonitor?channel={{channel}}&subtype={{stream}},H.264,554,0;1,0;1;2;3,,,,,admin,admin,,http://amcrest.com/fileuploader/download/download/?d=0&file=custom%2Fupload%2FAmcrest_NVR_User_Manual_V3.3.pdf
 Amcrest,camera,IP2M-841,(90:02:A9),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/cam/realmonitor?channel=0&subtype={{stream}},H.264,554,0;1,,,,1,0,admin,admin,,https://support.amcrest.com/hc/en-us/article_attachments/203939917/Amcrest_PT_Camera_User_Manual_v1.1.7.pdf
-GenIV,nvr,G4-HBRPRO8-1TB,(90:02:A9),rtsp://{{ip_address}}/cam/realmonitor?channel={{channel}}&subtype={{stream}}&authbasic={{#base64}}{{username}}:{{password}}{{/base64}},H.264,554,0;1,0;1;2;3;4;5;6;7,,,,,admin,admin,,http://www.g4direct.com/Help-Support/User-Manuals-and-Quick-Start-Guides
\ No newline at end of file
+GenIV,nvr,G4-HBRPRO8-1TB,(90:02:A9),rtsp://{{ip_address}}/cam/realmonitor?channel={{channel}}&subtype={{stream}}&authbasic={{#base64}}{{username}}:{{password}}{{/base64}},H.264,554,0;1,0;1;2;3;4;5;6;7,,,,,admin,admin,,http://www.g4direct.com/Help-Support/User-Manuals-and-Quick-Start-Guides
+Hanwha Techwin,camera,Hanwha,(00:09:18),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/profile2/media.smp,H.264,554,,,,,,,admin,4321,,
+Bosch,camera,Bosch,(00:04:63),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/rtsp_tunnel,H.264,554,,,,,,,,,,

```

Comment:
We should later remove the requirement for username, since Bosch doesn't require authentication by default. But for now, I entered an arbitrary default username of admin so that you could import @jhallard .